docs/source/cookbook/aggregated_node.rst

```diff
@@ -0,0 +1,89 @@
+Aggregated nodes
+----------------
+
+The `AggregatedNode` is a special node that allows constraints to be added to a model that apply to a group of (possibly disconnected) nodes, enforcing either a fixed ratio or fixed maximum.
+
+Constraining flow using a ratio
+===============================
+
+Aggregated nodes allows a constraint to be added that ensures the flow via two or more nodes conforms to a specific ratio. This is useful, for example, when modelling a blending constraint between multiple sources of water.
+
+In the example below the aggregated node "D" constrains the flow in nodes "A" and "B" to be equal (0.5 + 0.5 == 1). Due to the constraint the solution is flow from A = 40, B = 40, despite C demanding 100.
+
+Note that the aggregated node "D" is not connected to any nodes via an edge. The constraint is applied regardless.
+
+.. code-block:: javascript
+
+    "nodes": [
+        {
+            "name": "A",
+            "type": "input"
+        },
+        {
+            "name": "B",
+            "type": "input",
+            "max_flow": 40.0
+        },
+        {
+            "name": "C",
+            "type": "output",
+            "max_flow": 100.0,
+            "cost": -10.0
+        },
+        {
+            "name": "D",
+            "type": "aggregated",
+            "nodes": ["A", "B"],
+            "factors": [0.5, 0.5]
```

Comment:
We should comment that these ratios must (currently) be constant for an entire simulation.